aframe/pipelines/sandbox/configs/base.cfg

```diff
@@ -138,15 +138,6 @@ aframe_instances = 1
 preproc_instances = 1
 clean = true
 
-[luigi_Train]
```

Comment:
Why is the whole training section removed here?

Reply:
Ah, the reason is b/c the only difference between running the `aframe.pipelines.sandbox.Sandbox` and `aframe.pipelines.sandbox.Tune` pipelines is the name of the "train like" task. For the `Sandbox` you run `Train` and for `Tune` you run `TuneRemote`. So, to have it such that both pipeliens can inherit from this config, i specify the training config in the parent.